tests/security_test.go

```diff
@@ -382,3 +383,39 @@ func runSecretTest(secretName string, secretPath string, secretKey string, expec
 
 	return nil
 }
+
+func TestPodSecurityContext(t *testing.T) {
+
+	// run driver and executor as user "nobody"
+	const uid = "65534"
+	const gid = "65534"
```

Comment:
just wondering, can we use user and group names instead of UIDs?

Reply:
Looks like it's not possible. As per [k8s api reference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#securitycontext-v1-core), those fields must of type `integer`.